pyproject.toml

```diff
@@ -42,7 +43,7 @@ tests = [
     "tox",
     "sphinx",
     "furo",
-    "moto[s3,ssm]",
+    "moto[s3,ssm]>=4.0,<5.0",
```

Comment:
I'll look into this separately, but at a glance I did not see any deprecations in the change log for moto. That said, as it's a major version increment there may still be some so I'll dig into this deeper with my next set of Grove changes.